orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_runner.py

```diff
@@ -164,6 +164,13 @@ def register_context(result):
 
             # Extract results as DLPack tensors plus autograd context. Also skips all None values.
             unwrapped_values = wrap_all_outputs(result, is_training_mode)
+            ctx = unwrapped_values[0]
+            if ctx and ctx.saved_tensors:
+                for tensor in ctx.saved_tensors:
+                    if tensor in set(new_wrapped_args):
+                        raise wrap_exception(
+                            RuntimeError(f"When save input for backward, you need to clone and save its copy not itself.")
```

Comment:
Remove f if there is no variable to output as the pylint complains. 

Shall we give a more concrete info how to mitigate the issue? 

"PythonOp is saving input for backward, it's risky since ONNX Runtime backend is not aware of this, the tensor  buffer may be "released" by ORT, then potentially modified by other operators before backward function executes.
Try to mitigate this risk by changing 'ctx.save_for_backward(input)' to 'ctx.save_for_backward(input.detach().clone())'."

Reply:
thx, done.